src/Database/Sql.php

```diff
@@ -823,8 +741,13 @@ public function validateColumn(string $table, string $column): bool
 	 * @param bool $set If true builds a set list of values for update clauses
 	 * @param bool $enforceQualified Always use fully qualified column names
 	 */
-	public function values(string $table, $values, string $separator = ', ', bool $set = true, bool $enforceQualified = false): array
-	{
+	public function values(
+		string $table,
+		mixed $values,
+		string $separator = ', ',
+		bool $set = true,
+		bool $enforceQualified = false
+	): array {
```

Comment:
```suggestion
	public function values(
		string $table,
		mixed  $values,
		string $separator = ', ',
		bool   $set = true,
		bool   $enforceQualified = false
	): array
	{
```

This change (and followings) suggested with based on `.editorconfig` by PHPStorm IDE.